v0/destinations/active_campaign/transform.js

```diff
@@ -53,6 +54,9 @@ const responseBuilderSimple = (payload, category, destination) => {
   // fail-safety for developer error
   throw new CustomError("Payload could not be constructed", 400);
 };
+let endpoint;
```

Comment:
These variables should be scoped to methods

Reply:
Updated

Reply:
Scoped.